server/cmwell-bg/src/test/scala/cmwell/bg/test/CmwellBGSpec.scala

```diff
@@ -466,6 +467,58 @@ class CmwellBGSpec extends AsyncFunSpec with BeforeAndAfterAll with Matchers wit
       }
     }
 
+    val reProcessNotIndexedOWCommands = afterFirst{
+      val currentTime = System.currentTimeMillis()
+      val infotons = Seq.tabulate(5) {n =>
+          ObjectInfoton(
+            path = s"/cmt/cm/bg-test/re_process_ow/info_override",
+            dc = "dc",
+            indexTime = Some(currentTime + n*3),
+            lastModified = new DateTime(currentTime +n),
+            indexName = "cm_well_p0_0",
+            fields = Some(Map(s"a$n" -> Set(FieldValue(s"b$n"), FieldValue(s"c${n % 2}"))))
+          )
+      }
+      val owCommands = infotons.map{ i => OverwriteCommand(i)}
+
+      // make kafka records out of the commands
+      val pRecords = owCommands.map { writeCommand =>
+        val commandBytes = CommandSerializer.encode(writeCommand)
+        new ProducerRecord[Array[Byte], Array[Byte]]("persist_topic", commandBytes)
+      }
+
+      // send them all
+      val sendEm = Future.traverse(pRecords)(sendToKafkaProducer)
+
+      sendEm.flatMap { _ =>
+        scheduleFuture(5.seconds) {
+          ftsServiceES.deleteInfotons(infotons).flatMap { _ =>
+            scheduleFuture(5.seconds) {
+              val sendAgain = Future.traverse(pRecords)(sendToKafkaProducer)
+              scheduleFuture(5.seconds) {
```

Comment:
all the scheduling can be replaced with `skipCheck`s (see integration tests `Helpers` trait). it will:
- perform faster (5 seconds are probably too much, and we need 3 waits)
- be safer, and won't fail in case it takes longer than 5 seconds (due to e.g: gc)

Reply:
will be done as part of improving all BG tests